_posts/2022-11-10-CZI_EOSS5.md

```diff
@@ -3,4 +3,19 @@ layout: post
 title: MDAnalysis CZI EOSS 5 Grant: Outreach and Project Manager
 ---
 
-body
\ No newline at end of file
+We are happy to announce that MDAnalysis has been awarded a grant from the Chan Zuckerberg Initiative as part of the [Essential Open Source Software for Science][] program: “EOSS5: Growing the MDAnalysis community sustainably: A dedicated project manager, teaching and outreach initiatives”. 
+
+The MDAnalysis organisation strongly believes that engaging in outreach, mentoring and teaching is key to its mission of being the leading software package for molecular simulation analysis in Python.
+
+This 2-year grant will enable us to hire a full-time community, outreach, and project manager to extend our teaching and mentoring commitments and engage with the molecular simulation community across academia and industry.
+
+Over the next two years, our key deliverables include:
+- Increasing our participation in outreach activities
+- Organising 2 user group meetings (UGMs)
+- Hosting a series of online teaching workshops (3 per year)
+- Networking with other software projects in the molecular simulation space
```

Comment:
```suggestion
- Networking with other software projects within and outside the molecular simulation space
```
Kinda thinking if we need to work with file storage folks that might be outside mol sim?